src/frontend/app.ts

```diff
@@ -46,78 +54,79 @@ const oauth = GithubOAuth({
   githubClient: process.env.GITHUB_CLIENT_ID,
   githubSecret: process.env.GITHUB_CLIENT_SECRET,
   baseURL: process.env.APP_URL,
-  loginURI: '/github/login',
-  callbackURI: '/github/callback',
+  loginURI: "/github/login",
```

Comment:
Think we need to agree on linting. The prettierrc file you added sets quotes to single with a trailing comma. Linting changes in this PR change quotes to double and no trailing commas 😕 

<img width="441" alt="Screen Shot 2021-07-27 at 10 56 39 am" src="https://user-images.githubusercontent.com/37155488/127078068-afebc7f5-9a35-49a1-8cdd-24263add15c3.png">

Reply:
I'll create another PR to fix styling issues.